examples/arithmetic/uniffi.toml

```diff
@@ -1,2 +1,9 @@
 [bindings.kotlin]
 package_name = "org.mozilla.uniffi.example.arithmetic"
+cdylib_name = "arithmetical"
+
+[bindings.python]
+cdylib_name = "arithmetical"
+
+[bindings.swift]
+cdylib_name = "arithmetical"
```

Comment:
I think it would be nice to be able to specify this once, perhaps in a top-level `[bindings]` config group, but that's a bigger refactor of the way config works than I was willing to take on this afternoon.